<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseSearchWidget</class>
 <widget class="QWidget" name="DatabaseSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_search">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Search</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="combo_search_options"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_search_input"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_search">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../icons/icons.qrc">
            <normaloff>:/search_icon</normaloff>:/search_icon</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="ElementTableView" name="dataView_search_results">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ElementTableView</class>
   <extends>QTableView</extends>
   <header>GEMEditor.base.views</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
